peaks.js.d.ts

```diff
@@ -48,10 +48,7 @@ declare module 'peaks.js' {
     update: (options: PointUpdateOptions) => void;
   }
 
-  interface RequiredOptions {
-    /** HTML5 Media element containing an audio track */
-    mediaElement: Element;
-  }
+  interface RequiredOptions {  }
```

Comment:
I think we should have made `mediaElement` optional when we added the `player` option, so thanks for catching this. As `RequiredOptions` is now empty, we should remove it entirely.